src/android/SaveImageGallery.java

```diff
@@ -182,6 +182,8 @@ private File savePhoto(Bitmap bmp, String prefix, String format, int quality) {
                     folder.mkdirs();
                 }
 
+            } else if (deviceVersion.equals("11")) {
```

Comment:
Would it be better to check for equal or greater versions?

Reply:
It's a good point. For integrity I used `SDK_INT` as a comparison method.